CMakeLists.txt

```diff
@@ -532,6 +532,11 @@ else (WIN32)
 	set(CPACK_PACKAGE_ICON "${CMAKE_SOURCE_DIR}/docs/img/HELICS.ico")
 endif(WIN32)
 
+if (APPLE)
+    set(CPACK_BUNDLE_NAME "libhelics")
+    set(CPACK_BUNDLE_PLIST Info.plist)
+    set(CPACK_BUNDLE_ICON HELICS.ico)
```

Comment:
can these be set to the full path instead of needing to copy them over?

Reply:
Yes, I will do that.